cartographer_ros/cartographer_ros/node.h

```diff
@@ -75,16 +95,23 @@ class Node {
 
   cartographer::common::Mutex mutex_;
   MapBuilderBridge map_builder_bridge_ GUARDED_BY(mutex_);
-  int trajectory_id_ = -1;
-  std::unordered_set<string> expected_sensor_ids_;
 
   ::ros::NodeHandle node_handle_;
   ::ros::Publisher submap_list_publisher_;
   ::ros::ServiceServer submap_query_server_;
   ::ros::Publisher scan_matched_point_cloud_publisher_;
   cartographer::common::Time last_scan_matched_point_cloud_time_ =
       cartographer::common::Time::min();
-
+  ::ros::ServiceServer start_trajectory_server_;
```

Comment:
Yes, these need to live for the lifetime of the node just the same as other services(e.g. finish trajectory). Should I use a vector for all services?